protocol.html

```diff
@@ -730,13 +796,13 @@ <h2 property="schema:name">Notification Features</h2>
 
               <dl>
                 <dt about="#feature-startTime" property="skos:prefLabel" typeof="skos:Concept"><dfn id="feature-startTime">startTime</dfn></dt>
-                <dd about="#feature-startTime" property="skos:definition">An ISO 8601 datetime value indicating proposed or actual starting time of a subscription.</dd>
+                <dd about="#feature-startTime" property="skos:definition">A <code>xsd:dataTime</code> value indicating proposed or actual starting time of a subscription.</dd>
                 <dt about="#feature-endTime" property="skos:prefLabel" typeof="skos:Concept"><dfn id="feature-endTime">endTime</dfn></dt>
-                <dd about="#feature-endTime" property="skos:definition">An ISO 8601 datetime value indicating proposed or actual ending time of a subscription.</dd>
+                <dd about="#feature-endTime" property="skos:definition">A <code>xsd:dataTime</code> value indicating proposed or actual ending time of a subscription.</dd>
                 <dt about="#feature-state" property="skos:prefLabel" typeof="skos:Concept"><dfn id="feature-state">state</dfn></dt>
-                <dd about="#feature-state" property="skos:definition">An opaque value representing the last known state of a resource. If the resource state is known to have changed when the client establishes a subscription, an initial notification must be sent to the client. This value should correspond to a resource's <code>ETag</code> header value.</dd>
+                <dd about="#feature-state" property="skos:definition">A <code>xsd:string</code> value representing the last known state of a resource. If the resource state is known to have changed when the client establishes a subscription, an initial notification must be sent to the client. This opaque value should correspond to a resource's <code>ETag</code> header value.</dd>
                 <dt about="#feature-rate" property="skos:prefLabel" typeof="skos:Concept"><dfn id="feature-rate">rate</dfn></dt>
-                <dd about="#feature-rate" property="skos:definition">An ISO 8601 duration value indicating the minimum amount of time to elapse between notifications sent to the client.</dd>
+                <dd about="#feature-rate" property="skos:definition">A <code>xsd:duration</code> value indicating the minimum amount of time to elapse between notifications sent to receiver.</dd>
                 <dt about="#feature-accept" property="skos:prefLabel" typeof="skos:Concept"><dfn id="feature-accept">accept</dfn></dt>
                 <dd about="#feature-accept" property="skos:definition">A MIME-Type value that indicates the desired serialization of the notification. For instance, <code>text/turtle</code> may be acceptable.</dd>
```

Comment:
Accept should use a definition analogous to HTTP. In this case, content type in the form of MIME type (which is much more to the point than serialization).